pkg/docker/docker.go

```diff
@@ -800,15 +1011,18 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) error {
 			// the container exits normally).  dump port/etc information for the user.
 			dumpContainerInfo(container, d, image)
 		}
+
 		// Return an error if the exit code of the container is
 		// non-zero.
 		glog.V(4).Infof("Waiting for container %q to stop ...", containerName)
-		exitCode, err := d.client.WaitContainer(container.ID)
+		ctx, cancel := getDefaultContext((1<<63 - 1) * time.Nanosecond) // infinite duration ... go does not expost max duration constant
```

Comment:
also go does expose max int constants, you should probably use that instead of 1<<63 -1, though it is cute :)